spec/rollbar/sidekig/reset_scope_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'spec_helper'
```

Comment:
There was a misspelling in the pathname of spec/rollbar/sidekig/clear_scope_spec.rb that has been carried over to this renamed file. (sidekiq was spelled with a 'g' instead of a 'q'.)

This would be an ideal time to correct that spelling error.

Reply:
Addressed in 4fd0007